src/Raven.Server/Documents/Expiration/ExpirationStorage.cs

```diff
@@ -75,43 +75,54 @@ private void ThrowWrongExpirationDateFormat(Slice lowerId, string expirationDate
                 $"The expiration date format for document '{lowerId}' is not valid: '{expirationDate}'. Use the following format: {_database.Time.GetUtcNow():O}");
         }
 
-        public Dictionary<Slice, List<(Slice LowerId, string Id)>> GetExpiredDocuments(DocumentsOperationContext context,
-            DateTime currentTime, bool applyToExistingDocuments, long take, out Stopwatch duration, CancellationToken cancellationToken)
+        public class ExpiredDocumentsOptions
         {
-            return GetDocuments(context, currentTime, applyToExistingDocuments, DocumentsByExpiration, Constants.Documents.Metadata.Expires, take, out duration, cancellationToken);
+            public DocumentsOperationContext Context;
+            public DateTime CurrentTime;
+            public bool IsFirstInTopology; 
```

Comment:
This is from `applyToExistingDocuments` , no? Why the different name?

Reply:
@lastav5 any comment?